sql/api/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -54,17 +54,20 @@ abstract class RuntimeConfig {
   }
 
   /**
-   * Returns the value of Spark runtime configuration property for the given key.
+   * Returns the value of Spark runtime configuration property for the given key. If the key is
+   * not set yet, return `defaultValue` in its [[ConfigEntry]].
```

Comment:
I think real error is this:

```
```
[error] Generating /__w/spark/spark/target/javaunidoc/org/apache/spark/sql/RuntimeConfig.html...
[error] /__w/spark/spark/sql/api/target/java/org/apache/spark/sql/RuntimeConfig.java:13:1:  error: reference not found
[error]    * not set yet, return <code>defaultValue</code> in its {@link ConfigEntry}.
[error]                                                                  ^
[error] /__w/spark/spark/sql/api/target/java/org/apache/spark/sql/RuntimeConfig.java:25:1:  error: reference not found
[error]    * {@link ConfigEntry} is not the desired one.
[error]             ^
```
```

Let's just switch `[[ConfigEntry]]` to something else or remove it. `ConfigEntry` isn't technically an API so there is no way for the end users to see.

Reply:
Thanks @HyukjinKwon , I changed the wording and the CI is green now.